### DISCUSSION TITLE
glTF Anisotropy extension not supported?

### DISCUSSION THREAD

**Discussion by echadwick-artist** [google/model-viewer#4433] [Q&A] [answered]
When I create embeds using the code sample at https://modelviewer.dev/, glTF models with KHR_materials_anisotropy are not rendering the anisotropy effect.

First embed here uses anisotropy: https://ericchadwick.com/img/ecommerce.html

It uses this GLB: https://github.com/KhronosGroup/glTF-Sample-Assets/tree/main/Models/AnisotropyBarnLamp/glTF-Binary

My embed (left) versus the <model-viewer> Editor (right):
![2023-08-21 11_57_23-NVIDIA GeForce Overlay DT](https://github.com/google/model-viewer/assets/40177036/2035d739-ff21-4d7a-ae04-495a62e3977e)

HDRI is from Poly Haven: https://polyhaven.com/a/christmas_photo_studio_07

I also tried hosting copies of `model-viewer.min.js` and `model-viewer.min.js.map` and editing the js to increment to version 153 (n="153"). However that didn't work either, so I reverted back to the official URI.

**Comment by elalish** [comment, marked as answer]
Sorry, I forgot to update our homepage with our latest version. Try `https://ajax.googleapis.com/ajax/libs/model-viewer/3.2.0/model-viewer.js` instead. We definitely support anisotropy!

**Reply by echadwick-artist** [reply]
Works a treat. Thanks Emmett!
